apps/mongodb/deployers/mongo_statefulset.yml

```diff
@@ -31,12 +31,12 @@ spec:
        - name: mongo
          image: mongo
          command:
-           - mongod
-           - "--replSet"
-           - rs0
-           - "--smallfiles"
-           - "--noprealloc"
-           - "--bind_ip_all"
+           #- mongod
+           #- "--replSet"
+           #- rs0
+           #- "--smallfiles"
+           #- "--noprealloc"
+           #- "--bind_ip_all"
```

Comment:
Why these lines are commented?

Reply:
@gprasath We don't need these commands. We are letting mongodb to use the default configuration for sever initiation, so to perform our loadgen operation correctly.